### ISSUE TITLE
samtools reaheader not mentioned in the SYNOPSIS

### ISSUE THREAD

**Issue by valeriuo** [samtools/www.htslib.org#29]
There is a bottom link in the SEE ALSO section, but the command is not listed at the top of http://www.htslib.org/doc/samtools.html with all the others.

**Comment by jkbonfield** [comment]
View is in there twice too. :)

These have already been fixed in https://github.com/samtools/samtools/pull/1154 so I think it just needs the man pages regenerating online.  I thought I'd done this (or added a PR for it), but apparently not.

**Comment by jkbonfield** [comment]
I also think it would be nice if the online man page included links to the sub-command man pages in-situ as they're being listed in the COMMANDS section.  They SEE ALSO has links, as does the synposis.

There is custom code to spot these in the synposis section:

```
elsif ($synopsis && /^(\S+)\s+(\S+)/ && -e "doc/$1-$2.1.html") {
```

Perhaps we should be looking for .TP followed by .B _subcommand_ too so we can add them there as well.

**Comment by jmarshall** [comment]
I don't think I got around to pushing it to [manconvert](https://github.com/jmarshall/manconvert) yet but a while back I was making a companion script to facilitate spraying these hyperlinks around.

The other even better approach is to add some groff_man markup for it, that could be `.de`ed to nothing at the top of the file but that `manconvert` could easily recognise. I recently learnt [`.UR`/`.UE` are conventional for hyperlinks](https://www.man7.org/linux/man-pages/man7/groff_man.7.html).

**Comment by valeriuo** [comment]
> These have already been fixed in [samtools/samtools#1154](https://github.com/samtools/samtools/pull/1154) so I think it just needs the man pages regenerating online. I thought I'd done this (or added a PR for it), but apparently not.

Yes. It was more of a reminder to check for these issues, once we decide to make next release.
